src/Handler/Table/Export/ExportTableToFileHandler.php

```diff
@@ -79,10 +86,42 @@ public function __invoke(
             'TableExportToFileCommand.fileCredentials is required to be S3Credentials.'
         );
 
+        // validate exportOptions
         $requestExportOptions = $command->getExportOptions();
-        $columnsToExport = $requestExportOptions && $requestExportOptions->getColumnsToExport() !== []
-            ? ProtobufHelper::repeatedStringToArray($requestExportOptions->getColumnsToExport())
-            : [];
+        if ($requestExportOptions) {
+            $columnsToExport = $requestExportOptions->getColumnsToExport()->count() > 0
```

Comment:
proc je ten check potreba? `getColumnsToExport` vraci vzdy `RepeatedField` a `repeatedStringToArray` vraci vzdy pole

Reply:
hmm, asi by tam byt nemusel, kouknu...

Reply:
tak nemusel, je to pryc